_config.yml

```diff
@@ -9314,6 +9314,9 @@ principles:
 - id: fp-012-naming-conventions
   layout: principle
   title: Naming Conventions (principle 12)
+- id: fp-013-notification
+  layout: principle
+  title: Notification of Changes (principle 13)
```

Comment:
This is a bit of a problem now - the automatic site builder has added principle 13 without it being properly announced. @nataled can you make sure that it clearly says draft of the principle 13 page at the moment, so before #2270 is merged?

Reply:
Thank you!

Reply:
Alas, that won't help the problem. Like I said, the PR was indicated as a draft from the moment I created it, yet the automatic site builder still did the thing.

Reply:
The scripts that build the site only look at the `master` branch -- no other branches, not PRs, and PR branch status is not relevant. The `master` branch is the published/accepted/official version of all our docs, data, and code.

The `master` branch contains this file https://github.com/OBOFoundry/OBOFoundry.github.io/blob/master/principles/fp-013-notification.md which was created when PR #1714 was merged.

Reply:
@jamesaoverton so would the fix be to move that file to someplace else for now, or rename it?

Reply:
@nataled - this file does _not_ seem to be flagged as a draft! Please do so asap!

Sorry @jamesaoverton I think the rules around merging new principle content was not well defined. @erik-whiting it is important we add to the Website SOP that content about principles that are not yet completely approved should not be merged into master (or, which we can discuss, be flagged with a big DRAFT banner at the top)

Reply:
@matentzn I don't know how to flag a file as draft. I thought you were talking about the PR, which, as mentioned, was created as a draft. James indicates that doing so doesn't matter. I cannot rename the file, flag it in any way, or move the file. I'm at a loss to explain how it stayed where it was for two months with no issues until now.

Reply:
> I don't know how to flag a file as draft. I thought you were talking about the PR, which, as mentioned, was created as a draft.

Just add "DRAFT" in big letters at the top?

Reply:
@matentzn done. Still won't keep it from going public though, which is really what we're after.

Reply:
Great, thank you!